CONTRIBUTING.md

```diff
@@ -24,6 +24,11 @@ Tests are authenticated using [Application Default Credentials](https://develope
 
 Ensure you are logged in using `gcloud login` or set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to the path of your credentials file.
 
+To run the tests, you'll need to run `go get` with the `-t` option. From `go help get`:
```

Comment:
Let's just add `-t` to the command at the top?

It should probably also have `...` in it:

`go get -t -d github.com/GoogleCloudPlatform/golang-samples/...`

Reply:
About to commit:

To run the tests, you'll need to run 
`go get -t -d github.com/GoogleCloudPlatform/golang-samples/...` 

From `go help get`:

> The -t flag instructs get to also download the packages required to build
> the tests for the specified packages.

This looks clear to me.  Is this acceptable for the project?